src/FSharpVSPowerTools.Logic/Linting/LintOptionsPage.fs

```diff
@@ -0,0 +1,39 @@
+﻿namespace FSharpVSPowerTools.Linting
+
+open FSharpVSPowerTools
+open Microsoft.VisualStudio.Shell
+open System.ComponentModel
+open System.Runtime.InteropServices
+open System.Windows
+open System.ComponentModel.Composition
+open Microsoft.VisualStudio.OLE.Interop
+open System.Collections.Generic
+
+[<Guid("f0bb4785-e75a-485f-86e8-e382dd5934a4")>]
+type LintOptionsPage() =
+    inherit UIElementDialogPage()
+
+    let getProjectPaths () =
```

Comment:
This function is currently unused? I guess it's needed when we load and save configs.